docs/README.md

```diff
@@ -1,16 +1,18 @@
 ## Cellery Documentation
 
 
-- [Get Started](https://github.com/wso2-cellery/sdk) - Includes steps to install, set up Cellery runtime and build and 
+- [Get Started](https://github.com/wso2-cellery/sdk/blob/master/README.md) - Includes steps to install, set up Cellery runtime and build and 
 deploy your first Cell. 
 
-- [CLI Reference](https://github.com/wso2-cellery/docs/cli-reference.md) - List Cellery commands. 
+- [Cellery Architecture](https://github.com/wso2-cellery/sdk/blob/master/docs/cellery-architecture.md) - Cellery architecture and 
+Cell based architecture.
 
-- [Cell Specification](https://github.com/wso2-cellery/spec/blob/master/README.md) - Includes key concepts of Cellery
-and the language syntax.
+- [Cell Specification](https://github.com/wso2-cellery/spec/blob/master/README.md) - Key concepts of Cellery.
 
-- [Cellery Architecture](https://github.com/wso2-cellery/docs/cellery-architecture.md) - Describes high level 
-architecture of Cellery Mesh.
+- [Cell Reference](https://github.com/wso2-cellery/sdk/blob/master/docs/cell-reference.md) - How to code Cell (Cellery language 
+syntax).
 
-- [Cellery and Kubernetes](https://github.com/wso2-cellery/docs/cellery-architecture.md) - Describe how Cellery maps to 
+- [CLI Reference](https://github.com/wso2-cellery/sdk/blob/master/docs/cli-reference.md) - How to use CLI (Cellery commands). 
+
+- [Cellery and Kubernetes](https://github.com/wso2-cellery/sdk/blob/master/docs/cellery-and-kubernetes.md) - How Cellery works with 
```

Comment:
Is this new line intentional ? Check above line as well

Reply:
fixed